src/models/methods/submit.ts

```diff
@@ -0,0 +1,26 @@
+import { BaseRequest, BaseResponse } from "./baseMethod";
+
+export interface SubmitRequest extends BaseRequest {
+  command: "submit"
+  tx_blob: string
+  fail_hard?: boolean
+}
+
```

Comment:
Is the 'Sign-and-Submit Mode' deprecated? (Otherwise it might be another format we need an interface for) https://xrpl.org/submit.html#sign-and-submit-mode

Reply:
We're not supporting signing via rippled nodes, because it's discouraged (it can be insecure, the node can mess with your transaction before signing it).

Reply:
Aside from the node messing with your transaction; you'd potentially be sending your private key over the internet, which is generally not very safe.